presubmit.sh

```diff
@@ -84,6 +90,6 @@ cmake .. -G Ninja \
       -DGL_IS_SUPPORTED=${BUILD_OPENGL_TEST} \
       -DVULKAN_INCLUDE_DIR=${TOP}/Vulkan-Headers/include/ \
       -DVULKAN_LIB_DIR=${TOP}/Vulkan-Loader/build/loader/
-cmake --build . -j3 --config Release
+cmake --build . -j3 --config "${BUILD_CONFIG}"
```

Comment:
I'm not sure if this does what's intended.  Before this change, we'd expect all builds to be release builds, but [this comment](https://github.com/KhronosGroup/OpenCL-CTS/pull/1483#issuecomment-1241886877) makes me think they're actually all debug builds?

As far as I know, the build type is typically set in the configuration step using `-D CMAKE_BUILD_TYPE=Release`.

Reply:
I think you might be right.  It turns out that the "Ninja" generator _does not support_ multiple configurations using `--config` (as listed in CMAKE_CONFIGURATION_TYPES, and defaulting to CMAKE_DEFAULT_BUILD_TYPE if set, or otherwise the first configuration type in the list).  There is a special "Ninja Multi-Config" generator that supports multiple configurations, but it was added in CMake 3.17.  The version of CMake included in Ubuntu 20.04 is 3.16, so it is too old to use "Ninja Multi-Config".

It's probably best if we use `CMAKE_BUILD_TYPE=Release`.